src/setup_payload/AdditionalDataPayloadGenerator.h

```diff
@@ -0,0 +1,104 @@
+/**
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file provides a utility to generate Additional Data payload and its members
+ *      (e.g. rotating device id)
+ *
+ */
+
+#pragma once
+#include <core/CHIPError.h>
+#include <support/BitFlags.h>
+#include <system/TLVPacketBufferBackingStore.h>
+
+namespace chip {
+namespace RotatingDeviceId {
+static constexpr unsigned kRotatingDeviceIdCounterStringMaxLength = 10;
+static constexpr unsigned kRotatingDeviceIdHashSuffixLength       = 16;
+static constexpr unsigned kRotatingDeviceIdMaxLength = kRotatingDeviceIdCounterStringMaxLength + kRotatingDeviceIdHashSuffixLength;
```

Comment:
These are already namespaced in chip::RotatingDeviceId. All these constants should:

- Be moved to the implementation file unless they are used
- Have the `kRotatingDeviceId` suffix reduced to `k` since the namespacing is sufficient and explicit namespacing should sufficient.

Reply:
they're used in more than AdditionalDataPayloadGenerator.cpp
i changed their names